openassessment/xblock/submission_mixin.py

```diff
@@ -29,6 +29,12 @@ class SubmissionMixin(object):
 
     """
 
+    UPLOADABLE_IMAGE_MIME_TYPES = ['image/gif', 'image/jpeg', 'image/pjpeg', 'image/png']
+
+    UPLOADABLE_FILE_MIME_TYPES = ['application/pdf'] + UPLOADABLE_IMAGE_MIME_TYPES
+
+    FILE_EXT_BLACK_LIST = ['exe', 'msi', 'app', 'dmg']
```

Comment:
Let's add .com to this list. There are many more to consider:
http://www.howtogeek.com/137270/50-file-extensions-that-are-potentially-dangerous-on-windows/
http://pcsupport.about.com/od/tipstricks/a/execfileext.htm
http://www.file-extensions.org/filetype/extension/name/program-executable-files